spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/ComputePartitionStatsSparkAction.java

```diff
@@ -80,6 +88,16 @@ public Result execute() {
   }
 
   private Result doExecute() {
+    if (forceRefresh) {
+      LOG.info(
+          "Clearing the existing partition statistics for all snapshots to enforce force refresh");
+      UpdatePartitionStatistics clearStats = table.updatePartitionStatistics();
+      table
+          .partitionStatisticsFiles()
```

Comment:
Should do we remove the statistics file for the given snapshot and initiate statistics collection fresh(instead of incremental) for that given snapshot?
Do we also need to clear stats of other snapshots?